lib/rubocop/cop/sorbet/no_dynamic_contract_includes.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+#
+# In preparation for adding Sorbet to the codebase we must remove all dynamic imports as these
```

Comment:
Can we have a more generic description for this cop? "the codebase" don't have much sense in the concept of this open-source project.

Reply:
https://github.com/Shopify/rubocop-sorbet/pull/27/files#diff-4892c3bddcbe192b3a1dd30691e5d63eR3